src/main/java/meditracker/MediTracker.java

```diff
@@ -67,20 +66,16 @@ public void run() {
             Command command;
             try {
                 command = commandParser.getCommand();
-            } catch (ArgumentNotFoundException | DuplicateArgumentFoundException | CommandNotFoundException e) {
-                System.out.println(e.getMessage());
+            } catch (ArgumentNotFoundException | DuplicateArgumentFoundException | CommandNotFoundException |
```

Comment:
Maybe you can consider having a parent class `ArgumentException` and have these exceptions `extend` from it

Reply:
Opened #123 in regards to this. Thanks for this suggestion.